android/app/src/main/java/com/mattermost/MainApplication.java

```diff
@@ -4,6 +4,8 @@
 import android.util.Log;
 
 import com.facebook.react.ReactApplication;
+import com.psykar.cookiemanager.CookieManagerPackage;
```

Comment:
Good catch, actually that was the react-native link, let me take a look and see in detail whats going on here. i'll post back